clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -241,6 +241,42 @@ class CIRLoopOpLowering : public mlir::OpConversionPattern<mlir::cir::LoopOp> {
     return mlir::success();
   }
 
+  void makeYieldIf(mlir::cir::YieldOpKind kind, mlir::cir::YieldOp &op, mlir::Block *to,
+                   mlir::ConversionPatternRewriter &rewriter) const {
```

Comment:
Seems like the changes overall are violating 80 cols and format style. You can apply clang-format to your patches before submitting the PR (see more info in https://llvm.org/docs/CodingStandards.html#source-code-width), in general most LLVM coding standards should be followed.